sechub-doc/src/docs/asciidoc/documents/tutorials/02_create_new_pds_solution.adoc

```diff
@@ -0,0 +1,177 @@
+== Create a PDS solution for a product
+
+[NOTE]
+====
+The following user guide is an instruction create a PDS solution for a new module eg. a scan tool or a wrapper.
+====
+
+. Install docker compose as plugin https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-22-04[ubuntu22.04 install docker compose]
+
+. Create a new folder in sechub/pds-solutions/<your pds solution>
+
+. Copy following files/ folders from another PDS-solution: +
+The README.md, the 01-script, The docker folder, the docker compose yaml used in the 01-script and the env file (if your product requires environment variables are needed)
+
+. Rename all folders to fit your product name
+
+. Modify all files (Dockerfile, scripts, yaml, env ... ) to fit your product needs
+
+. Modify the sechub/sechub-pds-solutions/xray/docker/scripts/<your product>.sh +
+This script is executed when a job is transferred to your PDS solution you have to execute your product here. The script is referenced in your pds-config.json.
+
+. Modify your pds-config.json +
+You can add additional optional or required parameters. They have the form pds.productname.variablename and readable as environment variables like this PDS_PRODUCTNAME_VARIABLENAME +
+
+=== Test PDS standalone with pds-tools
+
+Requires: sechub-solution-shared/scripts/01-test-pds.sh +
+(example with licenseScan and tar file upload)
+
+. Start your PDS solution with the 01-start-single-docker-compose.sh
+
+. Export environment variables
++
+----
+export PDS_SERVER=https://localhost:8444; export PDS_USERID=admin; export PDS_APITOKEN="pds-apitoken"; export PDS_PRODUCT_IDENTFIER=PDS_YOURPRODUCT
+----
+
+. Download PDS-Tools from release page https://github.com/mercedes-benz/sechub/releases
+
+. Create a sechub.json like the following
++
+[source,json]
+----
+{
+  "apiVersion": "1.0",
+  "licenseScan": {
+    "use": [
+      "test-sources"
+    ]
+  },
+  "data": {
+    "binaries": [
+      {
+        "name": "test-sources",
+        "fileSystem" : {
+        "folders" : [ "some-binary" ]
+      }
+      }
+    ]
+  }
+}
+----
+
+. Create "some-binary" file you want to test your product
```

Comment:
I would use a code scan rather than a binary scan. In addition, I would provide a real world example repo which one can scan and get red findings. See the full effect.